src/resources/i18n/microsoft/Messages_fr_FR.xml

```diff
@@ -30,6 +30,7 @@
    <translation id="microsoft_office_license_detail_user_firstname" qtlid="23323">Prénom</translation>
    <translation id="microsoft_office_license_detail_user_lastname" qtlid="3818">Nom</translation>
    <translation id="microsoft_office_license_detail_user_status" qtlid="26443">Statut</translation>
+   <translation id="microsoft_office_license_detail_user_error" qtlid="226423">Une erreur est survenue :</translation>
```

Comment:
Did this exist before ? Because we can't create qtlids by ourselves

Reply:
Yes :) I searched in the translation tool to find that :)